tools/PublishModules.ps1

```diff
@@ -55,6 +55,7 @@ if ($repo -ne $null) {
 
 $resourceManagerRootFolder = "$packageFolder\$buildConfig\ResourceManager\AzureResourceManager"
 $publishToLocal = test-path $repositoryLocation
+
```

Comment:
@EmmaZhu No need for this change - if you get a chance, please revert it, but I won't hold up the PR for an extra space :-)